src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -112,8 +119,11 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
       issueReply using updatedData
   }
 
+  var isDone = false
+
   when (Done) {
     case Event(IssueReply, data) =>
+      isDone = true
```

Comment:
Ok good point - my impression was that each Cromwell instance was only running one workflow at a time. But I guess that probably false?